README.md

```diff
@@ -3,4 +3,72 @@
 [![Join the chat at https://gitter.im/cesium-ml/cesium](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/cesium-ml/cesium?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Build Status](https://travis-ci.org/cesium-ml/cesium.svg?branch=master)](https://travis-ci.org/cesium-ml/cesium)
 
-Please see [our website](http://cesium.ml) for installation instructions.
+## Summary
+Cesium allows for the application of machine learning methods to timeseries data by doing three things:
+
+1. Extracting features from timeseries data (i.e. turning timeseries into feature sets)
+2. Building models on the basis of these features. The model-building capabilities of Cesium are drawn from [scikit-learn](www.github.com/scikit-learn/scikit-learn).
+3. Using these models for prediction. 
+
+Here's an [example](http://cesium.ml/docs/examples/EEG_Example_output.html).
+
+This package is distinct from [Cesium Web](www.github.com/cesium-ml/cesium_web), which is a front-end web application that allows people to do all three of the above through a web browser by running computations through Cesium. 
+
+## Requirements:
+
+- numpy>=1.10.4
+- scipy>=0.16.0
+- scikit-learn>=0.17.0
+- pandas>=0.17.0
+- dask
+- toolz
+- xarray>=0.8.1
+- gatspy>=0.3.0
+- netCDF4
+- cloudpickle
+
+## Installation from binaries:
+- MacOS: `pip install cesium`
+- Ubuntu/Debian: `pip3 install cesium`
+- Bash on Ubuntu on Windows: After installing pip (`apt-get -y install python-pip`), run `pip3 install cesium`
+
+## Installation from source:
+
+1. Clone the repository `git clone https://github.com/cesium-ml/cesium.git`
+2. Enter the newly cloned repository.
+3. Run `pip install -e .`
+
+## License:
```

Comment:
Simply state that we are using the 3-clause BSD license, and that the full license may be found in LICENSE.txt